src/libraries/System.Private.Xml/src/System/Xml/XmlQualifiedName.cs

```diff
@@ -2,7 +2,9 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-// Warning: Do not port implementation of GetHashCode to Desktop. See notes in SecureStringHasher
+// Warning: Do not port implementation of GetHashCode to Desktop.
```

Comment:
```suggestion
// Warning: Do not port implementation of GetHashCode to .NET Framework.
```

Reply:
This line that refers to .NET Framework can be deleted. The risk that somebody would accidentally port this to .NET Framework is exactly zero.

Reply:
> exactly zero

:-)

Reply:
I never knew Jan to be such an optimist. :)